### PR TITLE
Cron Jobs

### PR DESCRIPTION
**Pull request by jchromik**
A suggestion for the documentation of cron jobs.

Please merge
[travis-api #225](https://github.com/travis-ci/travis-api/pull/225) and
[travis-web #464](https://github.com/travis-ci/travis-web/pull/464) first.

### CODE REVIEW

File: user/cron-jobs.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+[support@travis-ci.com](mailto:support@travis-ci.com?subject=Cron)**
+
+<div id="toc"></div>
+
+## Preparation
+
+Travis CI supports building projects time-controlled every day,
+week or month no matter if there was a push on this repository or not.
+This feature is not enables by default. Please contact us if you need it:
+
```
Comment (plaindocs) [inline user/cron-jobs.md]: Can we change this to something like 

    Travis CI lets you schedule builds at regular intervals independently of whether any commits were pushed to the repository.

File: user/cron-jobs.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+[support@travis-ci.com](mailto:support@travis-ci.com?subject=Cron)
+
+After activation, a section called "Cron Jobs" appears in your settings page.
+
```
Comment (plaindocs) [inline user/cron-jobs.md]: Maybe

    If this feature is enabled for your repository, there is a "Cron Jobs" settings tab on your Travis CI page.

File: user/cron-jobs.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Thereafter the cron job is displayed above:
+
+<figure>
+  <img alt="cron job created" src="/images/cron-created.png"/>
+</figure>
+
+## Deleting Cron Jobs
+
+Deleting is even easier than adding a cron job.
```
Comment (plaindocs) [inline user/cron-jobs.md]: This sentence adds nothing. ;-)

**Comment by Nef10** [comment]
@plaindocs both other PRs have been merged :)

File: user/cron-jobs.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+title: Cron Jobs
+layout: en
+permalink: /user/cron-jobs/
+---
+
+**Please note that cron jobs are not enabled by default.
+Please ask us to unlock it for your repository:
```
Comment (plaindocs) [inline user/cron-jobs.md]: Slightly picky here, but can we change to "Please ask us to unlock this feature for your repository" as "it" does not match "cron jobs"

File: user/cron-jobs.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+[support@travis-ci.com](mailto:support@travis-ci.com?subject=Cron)**
+
+<div id="toc"></div>
+
+## Preparation
+
+Travis CI lets you schedule builds at regular intervals independently of whether
+any commits were pushed to the repository.
+This feature is not enables by default. Please contact us if you need it:
```
Comment (plaindocs) [inline user/cron-jobs.md]: s/enables/enabled/

**Comment by jchromik** [comment]
@plaindocs fixed!

**Comment by plaindocs** [comment]
One more thing (sorry :-p), we should be able to keep the markdown a little cleaner by using this syntax instead of the html.

    Alt text

**Comment by jchromik** [comment]
@plaindocs should now be markdown-ish ;-)

**Comment by plaindocs** [comment]
Thanks for all of that!

**Pull request merged**